core/src/toast/pane/pane.tsx

```diff
@@ -67,3 +72,5 @@ ToastPane.types = {
 		iconCls: text.failureStrong,
 	} as ToastPaneType,
 };
+
+ToastPane.styles = [s.default, shadow.boxStrong, border.radius].join(" ");
```

Comment:
It's common for us to have another level to define the style. At the moment Pane only has 1 style, but maybe in the future there are mores, like even Tab component has "outset" and "flat"

I suggest to put this in another level, like ToastPane.styles.outset